config.json

```diff
@@ -186,6 +186,18 @@
       "values": [ "debug", "release", "checked" ],
       "defaultValue": "debug"
     },
+    "EnableProfileGuidedOptimization": {
+      "description": "Enables IBC profile optimizations if profile data are available.",
+      "valueType": "property",
+      "values": [ true, false ],
+      "defaultValue": false
+    },
+    "OptimizationDataDir": {
+      "description": "Sets the path where the build should look for IBC profile data.",
+      "valueType": "property",
+      "values": [],
+      "defaultValue": ""
+    },
```

Comment:
That's correct: no support, and we can remove them later.

My point is just that I tend to avoid adding them in the first place (using the ` -- /p:Foo=bar` syntax instead) to avoid the extra maintenance cost. I'm fine leaving these in the PR, just a note.